src/factories/BranchFactory.sol

```diff
@@ -0,0 +1,20 @@
+pragma solidity ^0.4.13;
+
+import 'ROOT/libraries/Delegator.sol';
+
+
+// FIXME: remove once this can be imported as a solidty contract
+contract Branch {
+    function initialize(address parentBranch, int256 parentPayoutDistributionHash);
+}
+
+
+contract BranchFactory {
+
```

Comment:
Style: I'm not personally a fan of empty lines after contract definition.  Though, if we _do_ go with this pattern I recommend a matching empty line before the contract closing `}` to satisfy my OCD.  :D